crypto/evp/digest.c

```diff
@@ -332,6 +333,10 @@ int EVP_DigestFinal_ex(EVP_MD_CTX *ctx, unsigned char *md, unsigned int *isize)
 int EVP_DigestFinalXOF(EVP_MD_CTX *ctx, unsigned char *md, size_t size)
 {
     int ret = 0;
+    OSSL_PARAM params[2] = {
+        OSSL_PARAM_size_t(OSSL_PARAM_DIGEST_KEY_XOFLEN, &size),
+        OSSL_PARAM_END
+    };
```

Comment:
Unfortunately, because `&size` issn't known at compile time, you will have to use `OSSL_PARAM_construct_size_t`